docs/sections/user_guide/cli/drivers/chgres_cube.rst

```diff
@@ -93,6 +93,11 @@ Its contents are described in depth in section :ref:`chgres_cube_yaml`. Each of
 
      $ uw chgres_cube run --config-file config.yaml --cycle 2023-12-15T18 --batch --dry-run
 
+* Specifying the ``--key-path`` flag follows a path of keys leading through the config to extract the driver's configuration block.
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/arg_key_path.yaml
+
```

Comment:
I added the `key_path` arg documentation here for a single driver. Once I get some feedback on how this looks, I can add it to all drivers. I also can factor all the information out to it's own section (not just the shared yaml) since this will be the same across all drivers.

Reply:
I think it would keep things DRYer if the explanatory text as well as the YAML example were include-able. For example, something like `key_path_nested.yaml`
```
foo:
  bar:
    driver:
      # driver config block
```
and `key_path_simple.yaml`
```
driver:
  # driver config block
```
and `key_path.rst`
```
The ``--key-path`` option can be used to navigate from the top of the config to the driver's configuration block. For example, specifying ``--key-path foo.bar`` with config

.. literalinclude:: shared/key_path_nested.yaml
.. highlight:: yaml

is equivalent to using config

.. literalinclude:: shared/key_path_simple.yaml
.. highlight:: yaml

without specifying ``--key-path``.
```
Then, I think, we could just add the single line
```
.. literalinclude:: ../../../../shared/key_path.rst
```
to each driver doc to get the explanatory text and nicely formatted YAML snippets. I'm not sure we have other examples of one include including another, so hopefully that's more-or-less right.

Does that proposed explanatory text make sense?